### ISSUE TITLE
Success is reported when running non-existing task

### ISSUE THREAD

**Issue by ferki** [RexOps/Rex#841]
```
$ rex ldhjf
[2015-11-27 10:20:07] <local> INFO - All tasks successful on all hosts
```

Previous behaviour was:
```
$ rex ldhjf
[2015-11-27 10:21:32] <local> ERROR - No task named 'ldhjf' found. Task names are case sensitive and the module delimiter is a single colon.
```

**Comment by kablamo** [comment]
I'm going to work on this after #778  is merged because they touch the same code.

**Comment by ferki** [comment]
#778 is merged now.

I think we should set the exit code to non-zero (to cross of one entry from the list on #400), and we can put back the `$taskname not found` error message (to (partially?) solve #729).

**Comment by kablamo** [comment]
I think it should exit if it finds a bad task name.  Thats not what it did before though.

**Comment by ferki** [comment]
Well, I'm pretty sure since 0.54.0 rex behaved like:
```bash
$ rex blah
[2015-12-07 14:33:11] <local> ERROR - No task named 'blah' found. Task names are case sensitive and the module delimiter is a single colon.
```
because I added it on bc8273ada9972ea5438c0d560ed7a42ad832e1a3 :)

**Comment by ferki** [comment]
Fixed on #854
